go.mod

```diff
@@ -15,7 +15,10 @@ require (
 	sigs.k8s.io/controller-runtime v0.12.2
 )
 
-require go.goms.io/fleet v0.6.1
+require (
+	github.com/stretchr/testify v1.7.0
```

Comment:
Originally this repo does not use any assert package but performs the comparison directly; considering the state of this repo I don't have a strong opinion on this though.